examples/research_projects/movement-pruning/emmental/modeling_bert_masked.py

```diff
@@ -29,8 +29,9 @@
 from emmental import MaskedBertConfig
 from emmental.modules import MaskedLinear
 from transformers.file_utils import add_start_docstrings, add_start_docstrings_to_model_forward
-from transformers.modeling_utils import PreTrainedModel, prune_linear_layer
+from transformers.modeling_utils import PreTrainedModel
```

Comment:
Research projects use a pinned version of Transformers, so you shouldn't update this file.

Reply:
Reverted